server/graphql/types/src/types/user.rs

```diff
@@ -30,7 +30,7 @@ impl UserStoreNode {
     }
 
     pub async fn name_id(&self) -> &str {
-        &self.user_store.store_row.name_id
+        &self.user_store.store_row.name_link_id
```

Comment:
Develop was broken... probably wanna link name to store row here to get correct name